src/modules/timepix3/Timepix3Instrument.h

```diff
@@ -13,14 +13,23 @@
 
 #include <common/readout/ess/ESSTime.h>
 #include <common/readout/ess/Parser.h>
+#include <common/reduction/Hit2DVector.h>
+#include <common/reduction/clustering/Hierarchical2DClusterer.h>
 #include <readout/DataParser.h>
 #include <timepix3/Counters.h>
 #include <timepix3/Timepix3Base.h> // to get Timepix3Settings
 #include <timepix3/geometry/Config.h>
-#include <timepix3/geometry/Geometry.h>
+#include <timepix3/geometry/Timepix3Geometry.h>
 
 namespace Timepix3 {
 
+struct TimepixHit {
+  uint32_t X;
+  uint32_t Y;
+  uint64_t TimeOfFlight;
+  uint16_t ToT;
+};
```

Comment:
I don't think this is necessary

Reply:
Indeed, there are no references in the source code to this object anymore. I will remove